src/charts/donut/examples/donut-pct-chart.js

```diff
@@ -26,6 +26,15 @@
  * <li>.tooltipFn(d)   - user defined function to customize the tool tip (optional)
  * <li>.centerLabelFn  - user defined function to customize the text of the center label (optional)
  * <li>.onClickFn(d,i) - user defined function to handle when donut arc is clicked upon.
+ * <li>.label          - object containing properties for external label (optional)
```

Comment:
Maybe this should be labelConfig or something more descriptive so that it is understood that this is not just a simple label.